tests/Phinx/Migration/AbstractMigrationTest.php

```diff
@@ -314,7 +314,7 @@ public function testPostFlightCheckSuccess()
         $table = $migrationStub->table('test_table');
         $table->addColumn("column1", "integer", ['null' => true])->create();
 
-        $this->assertTrue($migrationStub->postFlightCheck());
+        $this->assertNull($migrationStub->postFlightCheck());
```

Comment:
if it is is void, there is nothing to check for.

Reply:
Sure, but a test with no assertions will be marked as "risky". A PHP function will return null if there is no return statement.

Reply:
To be clear, I'll happily change it (or add a `$this->assertTrue(true);`) if you think it's better.

Reply:
The problem is that void methods must not be checked or tooling like PHPStan complains.
Can't we check anything else maybe?

Otherwise some blind assert would be better, yeah.

Reply:
Okay, I'll add a dummy assert.